src/trainer.py

```diff
@@ -121,29 +121,17 @@ def build_trainer(
         opt_params["warmup"] = 0.1
```

Comment:
A comment a couple of lines above mentions transformer. Double check that it still applies to BERT or GPT.

Reply:
Yep it has to do with BERT.